compiler/src/dmd/parse.d

```diff
@@ -4581,8 +4581,10 @@ class Parser(AST, Lexer = dmd.lexer.Lexer) : Lexer
             }
             else if (t.ty == Tfunction)
             {
+                /* @@@DEPRECATED_2.105.1@@@
```

Comment:
The version should say when to turn it into an error, 10 releases ahead by default, so 2.115

Reply:
I think @dkorpel meant to change the version in the deprecated tag:

```suggestion
                /* @@@DEPRECATED_2.115.1@@@
```